pkgs/top-level/python-packages.nix

```diff
@@ -4256,6 +4256,10 @@ let
     };
   };
 
+  # py3k disabled, see https://travis-ci.org/NixOS/nixpkgs/builds/48759067
+  graph-tool = if isPy3k then null
```

Comment:
`pkgs/development/python-modules/generic/default.nix` throws a meaningful error message when a "disabled" version of Python is used. IMHO, this is a good idea, because it gives users feedback about what they're doing wrong. It would be nice if your "home-grown" expression would behave the same way.